packages/tinybird/src/client.ts

```diff
@@ -0,0 +1,43 @@
+import * as z from "zod";
+import type { Tinybird } from "@chronark/zod-bird";
```

Comment:
Check out the [docs](https://github.com/chronark/zod-bird#ingesting-data). Perfect for use to have a tinybird schema validation.

Reply:
It could be cool if we store the tiny bird pipe / schema in the repo too